user_guide_src/source/general/common_functions.rst

```diff
@@ -106,13 +106,15 @@ Service Accessors
 
 .. php:function:: lang($line[, $args[, $locale]])
 
-    :param string $line: The line of text to retrieve
+    :param string $line: The language filename and the key of the text to retrieve.
     :param array  $args: An array of data to substitute for placeholders.
-    :param string $locale: Specify a different locale to be used instead of default one.
+    :param string $locale: Specify a different locale to be used instead of the current locale.
+    :returns: The text in the language file
+    :rtype: list<string>|string
```

Comment:
@kenjis , is that(`rtype`) a typo?

Reply:
However, I didn't see anything in [documentation](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/documentation.rst) or [sphinx](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html/). Please provide a reference link.

Reply:
No. It is for `Return type:`. See https://github.com/search?q=repo%3Abcit-ci%2FCodeIgniter%20:rtype:&type=code

Reply:
`rtype` is documented in python domain: https://www.sphinx-doc.org/en/master/usage/domains/python.html#info-field-lists

Reply:
Thank you @michalsn .